twitter_scraper.py

```diff
@@ -16,15 +16,18 @@ def gen_tweets(pages):
 
         while pages > 0:
             d = pq(r.json()['items_html'])
-
-            tweets = [tweet.text for tweet in d('.tweet-text')]
+            tweets = [tweet.text_content() for tweet in d('.tweet-text')]
             last_tweet = d('.stream-item')[-1].attrib['data-item-id']
 
             for tweet in tweets:
+                if 'http' in tweet:
```

Comment:
For [lines 23-25](https://github.com/georgeglessner/twitter-scraper/blob/4ce16ce14c495a55b1cf2151400ecf277b2d4770/twitter_scraper.py#L23-L25), how about this?

```py
# 'check this outhttp://from-http-to-https'
tweet = re.sub('http', ' http', tweet, 1)
# 'check this out http://from-http-to-https'
```

Just throwing suggestions.